se.principia_web.principia.yml

```diff
@@ -0,0 +1,32 @@
+app-id: se.principia_web.principia
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+rename-appdata-file: se.principia_web.principia.metainfo.xml
+rename-desktop-file: principia.desktop
+rename-icon: principia
+command: principia
+copy-icon: true
+finish-args:
+  - "--socket=x11"
+  - "--socket=pulseaudio"
+  - "--device=all"
+  - "--share=network"
```

Comment:
why does it need network ?

Reply:
it connects to network to upload and play community levels